db/schema.rb

```diff
@@ -117,6 +117,7 @@
     t.bigint "external_tenant"
     t.string "name"
     t.string "description"
+    t.jsonb "settings", default: {}
```

Comment:
Does the `jsonb` column not have a default of `{}`? Just curious.

Reply:
Unfortunately no, it defaults to `nil` which wreaks havoc. This way we're not always checking for `nil?` every time we manipulate the settings hash.